packages/manager-react-components/src/hooks/date/useFormattedDate.ts

```diff
@@ -0,0 +1,46 @@
+import { useTranslation } from 'react-i18next';
+
+export const defaultUnknownDateLabel = 'N/A';
+
+export enum DateFormat {
+  /**
+   * dd/MM/YYYY
+   */
+  compact = 'compact',
+  /**
+   * dd abbreviated month YYYY
+   */
+  display = 'display',
+  /**
+   * dd month YYYY
+   */
+  fullDisplay = 'fullDisplay',
+}
```

Comment:
It's just for my information. I imagine that you have already had a debate on this.

Why don't use : 

```suggestion
export type DateFormat = "COMPACT" | "DISPLAY" | "FULL_DISPLAY";
```

Reply:
We didn't have a debate on this but imo :
If you have to use the variable it's clearer to have an enum.

If you have only to compare to a property, having an union type is lighter.

But in this case, you will have to type :

`useFormattedDate({ dateString: 'something', format: DateFormat.xxx })